### ISSUE TITLE
Trying to create a VAE version

### ISSUE THREAD

**Issue by acerio98** [ThibaultGROUEIX/AtlasNet#36]
I'm trying to create a modified version of this model that is a variational autoencoder so I can perform random sampling using the learned weights, using a sum of the chamfer distance and KLD as the loss function. I've been running into trouble with getting good reconstructions with my implementation training on 1000 of the ShapeNet plane models, as all results tend to look the same no matter what input I give. I'm still trying to understand all the nuances of VAEs myself as well as parts of the AtlasNet architecture, but I just wanted to ask -- can you think of any reason why converting the architecture to a VAE _wouldn't_ work? Thanks.

**Comment by ThibaultGROUEIX** [comment]
I am more a GAN fan than a VAE fan myself mostly because they are simpler. I guess in your case, I would build this simple baseline : pretrain an autoencoder, then fix the weight and then train a GAN to generate vectors in the latent space. Let me know how your experiments go, you've sparked my interest :)

**Comment by DariaBogdan** [comment]
Hi, I'm interested in modifying it into vae too. Is there any progress made so far? Thanks.
